woocommerce/woocommerce-paypal-payments
# AXO: Add support for Block Checkout

### Description

This PR adds support Fastlane support to the Block Checkout.

### Steps to Test

1. Add a product to cart.
2. Go to a page (with the Checkout Block present).
3. Make sure Fastlane is available and works.

### Screenshots

|Before|After|
|-|-|
|![before](https://github.com/user-attachments/assets/b6e2e531-d935-4b16-9d6d-1b4451740db3)|![after](https://github.com/user-attachments/assets/3c880d35-ae16-4f9a-89e4-433f84a3692e)|
